qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -197,6 +197,66 @@ def process_sample_template(self, study, user, callback):
 
         callback((msg, msg_level, 'sample_template_tab', None, None))
 
+    def add_to_sample_template(self, study, user, callback):
+        """Process a sample template from the POST method
+
+        Parameters
+        ----------
+        study : Study
+            The current study object
+        user : User
+            The current user object
+        callback : function
+            The callback function to call with the results once the processing
+            is done
+
+        Raises
+        ------
+        HTTPError
+            If the sample template file does not exists
+        """
+        # If we are on this function, the argument "sample_template" must
+        # defined. If not, let tornado raise its error
+        sample_template = self.get_argument('sample_template')
+
+        # Define here the message and message level in case of success
+        msg = ("The samples in sample template '%s' have been "
+               "added" % sample_template)
+        msg_level = "success"
+        # Get the uploads folder
+        _, base_fp = get_mountpoint("uploads")[0]
+        # Get the path of the sample template in the uploads folder
+        fp_rsp = join(base_fp, str(study.id), sample_template)
+
+        if not exists(fp_rsp):
+            # The file does not exists, fail nicely
+            raise HTTPError(400, "This file doesn't exist: %s" % fp_rsp)
```

Comment:
Testing the GUI, I got to this error, and while it is accurate, as a user who is not an admin, this information is relevant but not presented anywhere. Is there any easy way to fix this?